### PR TITLE
Feature/mcrypt

### PR DESCRIPTION
**Pull request by dobbobolina**
- Formatting
 - Fix certbot
 - Remove apt loop in pip
 - Move mcrypt to its own task

### CODE REVIEW

**Comment by pajtai** [comment]
@dobbobolina why'd you remove package-lock? it'll get regenerated at the next npm i.... it is automatically removed in the npm published bundle:
https://docs.npmjs.com/files/package-lock.json

File: ansible/common/php-fpm/tasks/main.yml [hunk 1]
```diff
@@ -19,9 +19,6 @@
       - php{{ php.version }}-imagick
       - php{{ php.version }}-imap
       - php{{ php.version }}-mbstring
-      - '{{ "php" + php.version + "-mcrypt"
-              if (php.version is version_compare("7.2","<"))
-              else [] }}'
```
Comment (pajtai) [inline ansible/common/php-fpm/tasks/main.yml:24]: I was moderately fond of this conditional 😢

Comment (dobbobolina) [reply ansible/common/php-fpm/tasks/main.yml:24]: Yeah I respected it, but it was broken once switched to the array format, don't know why.

**Comment by dobbobolina** [comment]
> @dobbobolina why'd you remove package-lock? it'll get regenerated at the next npm i.... it is automatically removed in the npm published bundle:
> https://docs.npmjs.com/files/package-lock.json

I removed it because well, there was not one before....

**Pull request merged**